docs/reference/transform/apis/put-transform.asciidoc

```diff
@@ -261,3 +287,31 @@ When the {transform} is created, you receive the following results:
   "acknowledged" : true
 }
 ----
+
+The following {transform} uses the `latest` method: 
+
+[source,console]
+--------------------------------------------------
+PUT _transform/ecommerce_transform2
+{
+  "source": {
+    "index": "kibana_sample_data_ecommerce"
+  },
+  "latest": {
+    "unique_key": "customer_id",
```

Comment:
`unique_key` always needs to be an array, hence: `"unique_key": ["customer_id"]`